src/Pim/Bundle/DataGridBundle/Extension/Pager/PagerResolver.php

```diff
@@ -7,63 +7,48 @@
 use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
 
 /**
- * Resolver for pager : determine which pager should be used depending on the grid and the storage driver.
+ * Resolver for pager : determine which pager should be used depending on the grid.
  *
  * @author    Julien Janvier <julien.janvier@akeneo.com>
  * @copyright 2014 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
 class PagerResolver
 {
-    /** @var DatasourceSupportResolver */
-    protected $supportResolver;
-
     /** @var PagerInterface */
     protected $ormPager;
 
     /** @var PagerInterface */
-    protected $mongodbPager;
+    protected $dummyPager;
+
+    /** @var array */
+    protected $gridsWithDummyPager;
```

Comment:
dummyPageredGrids ? :) I find the word "with" weird in a propertyName I prefer a passive form ^^